src/containers/explore-page/components/desktop/ExplorePage.tsx

```diff
@@ -122,9 +134,13 @@ const ExplorePage = ({
       <Section
         title={messages.justForYou}
         subtitle={messages.justForYouSubtitle}
-        layout={Layout.TWO_COLUMN_DYNAMIC_WITH_LEADING_ELEMENT}
+        layout={
+          isUnderground
+            ? Layout.TWO_COLUMN_DYNAMIC_WITH_DOUBLE_LEADING_ELEMENT
```

Comment:
overall the remote var was super clean, except this special casing here. a bit scary. wonder if we could be more pragmatic about the arrangement based on the count length or something

Reply:
I'll sync with you about this on slack - I went through a number of different iterations of this lol